### PR TITLE
LibJS: Add the Number.{MAX, MIN}_VALUE constants

### PR DESCRIPTION
**Pull request by IdanHo**
This makes 53 new test262 test cases pass.

### CODE REVIEW

**Comment by Hendiadyoin1** [comment]
Is the Numer.min value supposed to be -Max or essentially Epsilon?

Make sure You use the correct one, last time I checked, the names of the floatingpoint numeric limits were confusing

**Comment by IdanHo** [comment]
> Is the Numer.MIN_VALUE supposed to be -Max or essentially Epsilon?

`The value of Number.MIN_VALUE is the smallest positive value of the Number type, which is approximately 5 × 10-324.`

**Comment by Hendiadyoin1** [comment]
Ok then never MINd

File: Userland/Libraries/LibJS/Runtime/CommonPropertyNames.h [hunk 1]
```diff
@@ -24,6 +24,8 @@ namespace JS {
     P(LOG10E)                                \
     P(LOG2E)                                 \
     P(MAX_SAFE_INTEGER)                      \
+    P(MAX_VALUE)                             \
+    P(MIN_VALUE)                             \
     P(MIN_SAFE_INTEGER)                      \
```
Comment (linusg) [inline Userland/Libraries/LibJS/Runtime/CommonPropertyNames.h:29]:
```suggestion
    P(MAX_VALUE)                             \
    P(MIN_SAFE_INTEGER)                      \
    P(MIN_VALUE)                             \
```

**Review by linusg — CHANGES_REQUESTED** [review]

File: Userland/Libraries/LibJS/Runtime/CommonPropertyNames.h (hunk 1 above)
Comment (IdanHo) [reply Userland/Libraries/LibJS/Runtime/CommonPropertyNames.h]: whoops, fixed!

**Pull request merged**